snap/info.go

```diff
@@ -236,6 +238,10 @@ func (app *AppInfo) SecurityTag() string {
 	return fmt.Sprintf("snap.%s.%s", app.Snap.Name(), app.Name)
 }
 
+func (app *AppInfo) EnvironmentFilePath() string {
```

Comment:
Argh.. except we have `Dir` in several places, and we don't want `DIrPath` too.

Can you please change this to just `EnvironmentFile`?  We should fix `WrapperPath` separately.